source/jormungandr/jormungandr/instance.py

```diff
@@ -712,21 +712,24 @@ def reap_socket(self, ttl):
         logger = logging.getLogger(__name__)
         now = time.time()
 
-        def _reap_sockets(sockets):
+        def _reap_sockets(connector, sockets):
             while True:
                 try:
                     socket, t = sockets.popleft()
```

Comment:
Can't you just do the following and remove entirely the `sockets` parameter from the function `_reap_sockets`?

```suggestion
                    socket, t = connector._sockets.popleft()
```